src/components/ButtonGroup/index.tsx

```diff
@@ -1,14 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import combine from 'util/combine';
-import GroupButton from './GroupButton';
+import GroupButton, { GroupButtonProps } from './GroupButton';
 
-const ButtonGroup = ({ buttons, onChange, ...restBtnGroup }) => (
+type ButtonModificationProps = {
+  key: string | number;
+  value: number;
+  onClick: any;
+};
+
+type CombinedButtonProps = GroupButtonProps & ButtonModificationProps;
+export interface ButtonGroupProps {
+  /**
+   * Each object in array renders a GroupButton, and (generally) passes all it's values through
+   * as props to GroupButton
+   */
+  buttons: CombinedButtonProps[];
+
+  /**
+   * Fires when any Button is clicked in ButtonGroup
+   */
+  onChange?: any;
+}
+
+const ButtonGroup = ({
+  buttons,
+  onChange,
+  ...restBtnGroup
+}: ButtonGroupProps): JSX.Element => (
   <div {...restBtnGroup} className="btngroup">
     {buttons.map(({ value, key, Icon, onClick, label, ...restBtn }, index) => (
       <GroupButton
         key={key || label}
-        onClick={onChange ? combine(() => onChange(label || value), onClick) : onClick}
+        onClick={onChange}
```

Comment:
We'll want to restore this line back which will fix the above linting errors and should fix the tests as well?

Reply:
yeah, it fixed both